reference/plugins.md

```diff
@@ -2,7 +2,170 @@
 layout: page
 title: Plugins
 ---
-We're working on building a way for you to extend Praxis. Please join the
-[development mailing
-list](https://groups.google.com/forum/#!forum/praxis-development) if you'd like
-to join that conversation.
+
+## Overview
+
+Plugins provide a means to cleanly add new functionality to Praxis, either by extending the core classes with additional features, or by registering callbacks to run during request handling.
+
+Presently, extensions to the following classes are supported by the plugin loader, along with the primary reason to extend them:
```

Comment:
I'd go with 'an example of a reason you might want to extend them' instead of 'primary reason to extend them'